.github/workflows/ci-cd.yml

```
@@ -19,98 +19,106 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
+        python-version: [ '3.8', '3.9', '3.10', '3.11', '3.12', '3.13' ]
```

let's list things on separate lines
```suggestion
        python-version:
        - 3.8
        - 3.9
        - >-
          3.10
        - 3.11
        - 3.12
        - 3.13
```